paseos/tests/attitude_disturbances_test.py

```diff
@@ -0,0 +1,254 @@
+"""Tests to see whether the attitude disturbance models work as intended"""
+
+import numpy as np
+import pykep as pk
+import sys
+
+sys.path.append("../..")
+import paseos
+from paseos import ActorBuilder, SpacecraftActor
+from paseos.utils.reference_frame_transfer import eci_to_rpy, rpy_to_body
+from paseos.utils.load_default_cfg import load_default_cfg
+
+
+def test_gravity_disturbance_cube():
+    """This test checks whether a 3-axis symmetric, uniform body (a cube with constant density, and cg at origin)
+    creates no angular acceleration/velocity due to gravity. The spacecraft is initially positioned with the z-axis
+    aligned with the nadir vector."""
+    earth = pk.planet.jpl_lp("earth")
+
+    # Define local actor
+    sat1 = ActorBuilder.get_actor_scaffold("sat1", SpacecraftActor, pk.epoch(0))
+    ActorBuilder.set_orbit(sat1, [7000000, 0, 0], [0, 8000.0, 0], pk.epoch(0), earth)
+    ActorBuilder.set_spacecraft_body_model(sat1, mass=100)
+    ActorBuilder.set_attitude_model(sat1)
+    ActorBuilder.set_attitude_disturbances(sat1, gravitational=True)
+
+    # init simulation
+    sim = paseos.init_sim(sat1)
+
+    # Check initial conditions
+    assert np.all(sat1._attitude_model._actor_angular_velocity == 0.0)
+
+    # run simulation for 1 period
+    orbital_period = 2 * np.pi * np.sqrt((6371000 + 7000000) ** 3 / 3.986004418e14)
+    sim.advance_time(orbital_period, 0)
+
+    # check conditions after 1 orbit
+    assert np.all(np.round(sat1._attitude_model._actor_angular_acceleration, 10) == 0.0)
```

Comment:
only checks if the values are 0, not an actual correctness check with non-zero values